packages/mattermost-redux/src/selectors/entities/teams.ts

```diff
@@ -33,8 +34,28 @@ export function getTeams(state: GlobalState): IDMappedObjects<Team> {
     return state.entities.teams.teams;
 }
 
-export function getTeamsInPolicy(state: GlobalState): IDMappedObjects<Team> {
-    return state.entities.teams.teamsInPolicy;
+export function getTeamsInPolicy() {
+    return (createSelector(
+        getTeams,
+        (state: GlobalState, props: {policyId: string}) => getDataRetentionCustomPolicy(state, props.policyId),
+        (allTeams, policy) => {
+            if (!policy) {
+                return [];
+            }
+
+            const policyTeams: Team[] = [];
+
+            Object.entries(allTeams).forEach((item: [string, Team]) => {
+                const [, team] = item;
```

Comment:
I dont get the `,`.

Also, `item` probably should be `team`? 

also what does `item: [string, Team]` mean? Not familiar with that extrapolation.

Reply:
`item` isn't a channel, it's an array containing a string at the first index and team in the second index. `item: [string, Team]` is just showing the `item`'s type.